response_operations_ui/views/collection_exercise.py

```diff
@@ -105,7 +105,15 @@ def view_collection_exercise(short_name, period):
     ]
 
     ce_state = ce_details["collection_exercise"]["state"]
-    show_set_live_button = ce_state in ("READY_FOR_REVIEW", "FAILEDVALIDATION")
+    if "surveyMode" in ce_details["survey"] and ce_details["survey"]["surveyMode"] == "EQ":
```

Comment:
This can be replaced with:
```python
if ce_details["survey"].get("surveyMode") == "EQ"
```

`.get(key)` returns None if the key doesn't exist so you can write it with only 1 condition instead of having the `and`

Reply:
cheers.
It was the .get() that caused the failure as (null & null) returned the wrong boolean